### DISCUSSION TITLE
UX: following tags

### DISCUSSION THREAD

**Discussion by grahamperrin** [mastodon/mastodon#21838] [Q&A]
### Advanced interface

The result of my first follow of a tag: 

![image](https://user-images.githubusercontent.com/192271/204456220-abd243cc-71f3-4572-b5bf-2f2d31d732be.png)

* content becomes too wide for the window
* cropping, to the right. 

I could use horizontal scrolling, but it's not good UX. 

Is there some other UI to the tag, that does not involve widening? 

How can I remove, or hide, the column?

I can use a browser, instead of Mastodon, to bookmark <https://bsd.network/tags/freebsd>, but that's NLI (not logged in). 

### Non-advanced

I tried temporarily disabling the advanced interface. In the Twitter-like interface, I could not find a record of me following `#freebsd`. 

What am I missing?

### Background

Spun off from <https://bsd.network/@grahamperrin/109425668984125557>

### Environment

v3.5.5+glitch at https://bsd.network/

**Comment by ellotheth** [comment]
Following tags is a new feature in 4.0, and it adds tagged posts to your _Home_ timeline. `bsd.network` uses the [glitch fork](https://github.com/glitch-soc/mastodon/) of version 3.5.5, so tag following is not available. (You've pinned a tag to your advanced interface, which gives you a new column with tagged posts as you've seen, but won't add tagged posts to your _Home_ timeline.)

**Reply by grahamperrin** [reply]
Thanks. 

The **move** button does not work in this context: 

![image](https://user-images.githubusercontent.com/192271/204663450-dd69de6b-048b-4905-b2a6-f3d2a983eb0f.png)

– so for now, I'll unpin the column. 

When no longer pinned, does it remain easily accessible anywhere, without resorting to NLI?
